WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -155,19 +153,33 @@ def execute( self ):
         return self.__finish( 'Filling Mode is Disabled' )
 
     self.log.verbose( 'Job Agent execution loop' )
-    available = self.computingElement.available()
-    if not available['OK'] or not available['Value']:
+    result = self.computingElement.available()
+    if not result['OK'] or not result['Value']:
       self.log.info( 'Resource is not available' )
-      self.log.info( available['Message'] )
+      self.log.info( result['Message'] )
       return self.__finish( 'CE Not Available' )
 
-    self.log.info( available['Message'] )
+    self.log.info( result['Message'] )
+
+    ceInfoDict = result['CEInfoDict']
```

Comment:
Indeed. This is since looooong time before the S_OK was really standardized. Used in too many places to change everything in this PR, I think